framespacer/R/asdataframe.R

```diff
@@ -0,0 +1,12 @@
+#' FrameSpace DataFrame to R DataFrame
+#' @description Converts a FrameSpace DataFrame into an R DataFrame
+#' @param dataframe \strong{list} FrameSpace DataFrame
+#' @return R DataFrame
+#' @export
+as.dataframe <- function(dataframe){
```

Comment:
That's cool I'll play with it. I just have ran into a situation where I had to do something like `packageA::functionX` or `packageB::functionX` to distinguish and wanted to avoid that here. Thanks for the suggestion.